robottelo/ui/base.py

```diff
@@ -686,6 +686,11 @@ def select(self, target, list_value, wait_for_ajax=True, timeout=30,
         if self.element_type(target) == 'select':
             if isinstance(target, tuple):
                 element = self.wait_until_element(target)
+                if element is None:
```

Comment:
+StaleElementException is not the same as `UINoSuchElementError` and better to not be handled, but fixed in the code directly
Do not see any reason to mask initial issue.